README.md

```diff
@@ -69,17 +69,18 @@ application restarts.
 
 ### Use correct application data directories
 
-Modify Qt application main function by setting the organization and application name
-to match the declaration added to the desktop file.
+Applications that are not based on the Sailfish App library
+([libsailfishapp](https://github.com/sailfishos/libsailfishapp))
```

Comment:
Would it not make more sense to say apps that are not using the Sailfish App library instead of being based on it?

Reply:
Fixed